### ISSUE TITLE
Me, Interrupted — Desktop application is not reacting anymore to alphanumeric's delete key

### ISSUE THREAD

**Issue by x80486** [standardnotes/forum#1771]
I think the title says it all: the <kbd>delete</kbd> key stopped working in the last (or so) release — quite annoying this one :grimacing:

I'm under `Linux` and using `GNOME`.

**Comment by atmoio** [comment]
Do you mean desktop app?

**Comment by x80486** [comment]
That was certainly the case :+1:

**Comment by atmoio** [comment]
I'm assuming you're referring to the delete key with respect to the plain text editor? Does the delete key work for you in other text input boxes, like the search box or note title box?

**Comment by x80486** [comment]
It really doesn't work across the board — which is a good thing to replicate. I didn't test before in other parts, but now I did, and it doesn't behave as it used to.

Notice this is the <kbd>delete</kbd> on the alphanumeric part of the keyboard. The other <kbd>delete</kbd> key (what most people use in my experience) works as it should.

Interestingly enough, when <kbd>Num Lock</kbd> is active, that key will present a dot instead, and that works.

**Comment by atmoio** [comment]
Can you try other Electron based applications to see if the behavior is perhaps similar? I believe Slack and VSCode are Electron-based, to name a few.

**Comment by adrihut** [comment]
Wasn't able to replicate on my end @x80486. If the issue isn't present in other [electron apps](https://www.electronjs.org/apps), please clarify the following details for us, to get a better sense of what may be going on:

- What distro are you using?
- What installation method are you using for SN—*AppImage, Snap, etc.*?
- Are you experiencing this behavior with every **note type** (editor)?
- Is [SN-web](https://app.standardnotes.com/) working as expected on the same system?
- Have you observed the same behavior with SN on a different system?

**Comment by x80486** [comment]
I just tested `Slack`, `Bitwarden`, and `Postman` — on their latest versions. It works fine.

I'm using `Arch Linux`, and the Flatpak version of `SN`. The Web version (using `Brave` and probably any other supported browser) works fine.

I did test the `AppImage` in a VM I have with `Ubuntu` and it works fine...but I can't use the `AppImage` in `Arch Linux`. I'll move both issues to the `Flahub` repository because surprisingly enough, based on what I've tested, looks like they are related with the `Flatpak` only.
